query/sources/impl/src/main/java/org/codice/ddf/admin/sources/opensearch/OpenSearchSourceUtils.java

```diff
@@ -140,36 +140,33 @@ public OpenSearchSourceUtils(ConfiguratorSuite configuratorSuite) {
      *
      * @param responseField The URL to probe for OpenSearch capabilities
      * @param creds         optional credentials used in the original capabilities request
-     * @return a {@link ReportWithResultImpl} containing the {@link OpenSearchSourceConfigurationField} or containing {@link org.codice.ddf.admin.api.report.ErrorMessage}s on failure.
+     * @return a {@link Report} containing the {@link OpenSearchSourceConfigurationField} or containing {@link org.codice.ddf.admin.api.report.ErrorMessage}s on failure.
      */
-    private ReportWithResultImpl<OpenSearchSourceConfigurationField> getOpenSearchConfigFromResponse(
+    private Report<OpenSearchSourceConfigurationField> getOpenSearchConfigFromResponse(
             ResponseField responseField, CredentialsField creds) {
-        ReportWithResultImpl<OpenSearchSourceConfigurationField> configResult =
-                new ReportWithResultImpl<>();
 
         String responseBody = responseField.responseBody();
         int statusCode = responseField.statusCode();
 
         if (statusCode != HTTP_OK || responseBody.length() < 1) {
-            configResult.addArgumentMessage(unknownEndpointError(responseField.requestUrlField()
+            return Reports.from(unknownEndpointError(responseField.requestUrlField()
                     .path()));
-            return configResult;
         }
 
         Document capabilitiesXml;
         try {
             capabilitiesXml = sourceUtilCommons.createDocument(responseBody);
         } catch (Exception e) {
             LOGGER.debug("Failed to read response from OpenSearch endpoint.");
-            configResult.addArgumentMessage(unknownEndpointError(responseField.requestUrlField()
+            return Reports.from(unknownEndpointError(responseField.requestUrlField()
                     .path()));
-            return configResult;
         }
 
         XPath xpath = XPathFactory.newInstance()
                 .newXPath();
         xpath.setNamespaceContext(SOURCES_NAMESPACE_CONTEXT);
 
+        Report<OpenSearchSourceConfigurationField> configResult = Reports.emptyReport();
```

Comment:
We shouldn't need this if we return on the 3 below results where we do `Reports.from(...)`.